ci/builders/mac_host_engine.json

```diff
@@ -106,7 +104,6 @@
                     "base_path": "out/host_release/zip_archives/",
                     "type": "gcs",
                     "include_paths": [
-                        "out/host_release/zip_archives/darwin-x64-release/FlutterMacOS.framework.zip",
```

Comment:
shall we remove the `"flutter/build/archives:macos_flutter_framework",` in ninja targets as well

Reply:
The target is required to generate the files that will be later packaged.

Reply:
tru, yeah looks like it was updated at https://github.com/flutter/engine/commit/00a48acb208dc1daa4155932596f4096ca46af75